backend/game/dice_handler.py

```diff
@@ -0,0 +1,25 @@
+import game.dice.dice as dice
+
+
+class DiceHandler:
+    def __init__(self):
+        self.dice_values = []
+        self.re_rolls_left = 0
+
+    def start_turn(self, starting_dice_count, re_roll_count):
+        self.dice_values = dice.roll_many(starting_dice_count)
+        self.re_rolls_left = re_roll_count
+
+    def re_roll_dice(self, indexes_of_dice_to_re_roll):
+        if self.re_rolls_left > 0:
```

Comment:
An earlier point that was made was that we can return out here if it's the case that it's zero or less.

Reply:
this used to be 
```python
if self.re_rolls_left < 0
    return
```

but I thought Gerardo said the return muddled the purpose of the method. Maybe I'm misunderstanding.

Reply:
switched it back